homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -146,6 +182,14 @@ def alarm_trigger(self, code=None):
         """Send alarm trigger command."""
         raise NotImplementedError()
 
+    def alarm_keypress(self, keypress=None):
+        """Send custom key sequence to alarm."""
+        pass
```

Comment:
Thanks for your review - Just on recommendation in pull request 3884 where this originally started - so the other alarm services aren't required to implement it